### PR TITLE
updated the section navigation,created invetory index where every cop…

### PR DESCRIPTION
**Pull request by CodrAyush**
inventoryIndex main page have been created where i have called section navigation and section navigation im calling all the componnents.
in the app.jsx im calling inventoryIndex which u can open directly by 
http://localhost:5173/inventory
here are the folder structures:
image

how its looks like after this
image
image
image

### CODE REVIEW

File: src/Modules/Inventory/components/SectionNavigation.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+import { Group, Text, Box, Container } from "@mantine/core";
+import { CaretLeft, CaretRight } from "@phosphor-icons/react";
+import InventoryDashboard from "./inventoryDashboard";
+import Reports from "./Reports";
+
+// Import InventoryDashboard (ensure the path is correct)
+
+const sections = ["Overall Inventory", "Hostel", "Reports"];
```
Comment (deveshidwivedi) [inline src/Modules/Inventory/components/SectionNavigation.jsx:9]: Add Btech, Bdes in sections

**Comment by deveshidwivedi** [comment]
Add margin at the bottom here: 
image

File: src/App.jsx [hunk 2]
```diff
@@ -9,8 +9,8 @@ import LoginPage from "./pages/login";
 import ForgotPassword from "./pages/forgotPassword";
 import AcademicPage from "./Modules/Academic/index";
 import ValidateAuth from "./helper/validateauth";
-import InventoryDashboard from "./Modules/Inventory/components/inventoryDashboard";
 import Reports from "./Modules/Inventory/components/Reports";
+import InventoryIndex from "./Modules/Inventory/InventoryIndex";
```
Comment (deveshidwivedi) [inline src/App.jsx:13]: all components should be in inventory/components

File: src/Modules/Inventory/components/SectionNavigation.jsx (hunk 1 above)
Comment (CodrAyush) [reply src/Modules/Inventory/components/SectionNavigation.jsx:9]: this will be in overall inventory so we will be calling those in that

Comment (deveshidwivedi) [reply src/Modules/Inventory/components/SectionNavigation.jsx:9]: we still need them here

Comment (CodrAyush) [reply src/Modules/Inventory/components/SectionNavigation.jsx:9]: section is inly three over there

File: src/App.jsx (hunk 2 above)
Comment (CodrAyush) [reply src/App.jsx:13]: yes all componts are inventory/components
InvetoryIndex is just for landing page that why i have kept it outrside

Comment (CodrAyush) [reply src/App.jsx:13]: to make it look clear

File: src/Modules/Inventory/components/SectionNavigation.jsx (hunk 1 above)
Comment (CodrAyush) [reply src/Modules/Inventory/components/SectionNavigation.jsx:9]: ok mai add kr de rha but abhi componets bnne nhi hai to default text dal de rha

**Pull request merged**